document-generation/smartdocuments/src/test/kotlin/com/ritense/smartdocuments/plugin/SmartDocumentsPluginIntegrationTest.kt

```diff
@@ -170,6 +161,55 @@ class SmartDocumentsPluginIntegrationTest : BaseSmartDocumentsIntegrationTest()
         )
     }
 
+    @Test
+    fun `should resolve template-name value from process-variable`() {
```

Comment:
Do we want to include a test to validate how Valtimo responds when the process variable can't be found?